app/models/meetup.d.ts

```diff
@@ -0,0 +1,36 @@
+export interface MeetupEvent {
```

Comment:
Thanks for adding all of these types and so much organization! Mind just making this `meetup.ts`? We don't really need it to be a type definition file and I think it could just cause confusion down the line for other devs.

Reply:
that was on purpose in order to not cause confusion. a `.d.ts` indicates there's only types in this file. however, this point is moot simply b/c the process of adding yup-based parsing added more than types to this file so i renamed it accordingly.

Reply:
That makes sense. Yeah, I think my thought process is just that I often I'll end up adding more than just types to a file that starts as only types, so starting with `.ts` is one less thing to think about in the future.

Also, types declared in `.d.ts` files makes the types global, which is generally not what we want to do
![image](https://user-images.githubusercontent.com/12396812/206715349-15bbecb6-dcca-4324-a3d0-b641c6c6964e.png)

Reply:
huh. learn something new everyday. wasn't aware of the global side effect.
> If you want an interface/type that you can simply use without having to import it in the consuming module said interface must reside in a .ts or ideally a .d.ts file **_without any imports or exports being present in the same file._** This is of utmost importance because as soon as you are exporting at least one thing from the same file it becomes a module and everything that is in that module must be subsequently imported by consumers.

https://stackoverflow.com/q/42233987

so my previous `.d.ts` would've been a module since i was exporting things and thus no global types. but admittedly, that was on accident. 😅 such a unintuitive behavior choice from TS. now that i know that, i'll probably use `.d.ts` far less often. 

but yea, the parsing made this point moot. but thanks for sending me down that rabbit hole on odd TS behavior.